src/solver/optimisation/opt_rename_problem.h

```diff
@@ -0,0 +1,141 @@
+#pragma once
+#include "opt_structure_probleme_a_resoudre.h"
+#include "opt_export_structure.h"
+
+const std::string SEPARATOR = "::";
+const std::string AREA_SEP = "$$";
+
+class TargetVectorUpdater
+{
+public:
+    TargetVectorUpdater(bool isRenamingProcessed, std::vector<std::string>& target) :
+     target_(target)
+    {
+        if (isRenamingProcessed)
+        {
+            UpdateTargetAtIndex = std::bind(
+              &TargetVectorUpdater::UpdateTargetAtIndexImpl, this, std::placeholders::_1, std::placeholders::_2);
+        }
+    }
+
+    std::function<void(const std::string&, unsigned index)> UpdateTargetAtIndex
+      = [](const std::string&, unsigned index) {};
```

Comment:
`[](const std::string&, unsigned index) {};` : **unsigned** is a type ? Do you mean **unsigned int** ?

Reply:
unsigned = unsigned int